indico/modules/events/management/templates/event_protection.html

```diff
@@ -1,11 +1,12 @@
 {% extends 'events/management/base.html' %}
-{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+{% from 'forms/_form.html' import form_header, form_footer, form_row, form_rows, form_fieldset %}
 
 {% block title %}{% trans %}Event protection{% endtrans %}{% endblock %}
 
 {% block content %}
     {{ form_header(form, id='event-protection-form') }}
-    {{ form_rows(form, skip=form.priv_fields,
+    {{ form_row(form.permissions, skip_label=true) }}
+    {{ form_rows(form, skip=('priv_fields', 'permissions'),
```

Comment:
That doesn't do what you think it does!

`form.priv_fields` is a set containing the names of fields to skip. So you want `skip=form.priv_fields.union([permissions])` which is super ugly, but you don't have access to the `|` operator in Jinja.

Reply:
Why isn't the way I'm doing valid? I guess it's just a matter of the stuff to skip into `priv_fields`?

Reply:
Because there is no field named `priv_fields` - this is a set containing field names! So with your code, it skips `permissions` and that's it. since there is no field named `priv_fields`. But a conference event has extra fields for session coordinator privileges.